docs/source/api/core/iod.rst

```diff
@@ -0,0 +1,4 @@
+IOD Module
```

Comment:
This module contains no documented functions, I would remove the page or add a `TODO`.